app/src/main/java/com/example/erabook/fragments/discoverGoogle/SharedGoogleBooksViewModel.kt

```diff
@@ -22,10 +30,13 @@ class SharedGoogleBooksViewModel @Inject constructor(
     private val googleBooksApi: GoogleApi
 ) : ViewModel() {
 
+    private val db = Firebase.firestore
     private val _response_books = MutableLiveData<Resource<GoogleBooks>?>()
     val response_books: LiveData<Resource<GoogleBooks>?> = _response_books
     private val _loading_books = MutableLiveData<Boolean>()
     val loading_books: LiveData<Boolean> = _loading_books
+    private val _message = MutableLiveData<Boolean>()
+    val message: LiveData<Boolean> = _message
```

Comment:
This naming doesn't look like a boolean variable. Usually it's a good practice to name boolean variables with prefix: is, can, should! i.e isEnabled, canProceed, shouldMove etc.